cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionMetadata.java

```diff
@@ -28,12 +29,12 @@
 public class PartitionMetadata implements Iterable<Map.Entry<String, String>> {
   private final Map<String, String> metadata;
   private final long creationTime;
-  private final long lastUpdatedTime;
+  private final long lastModifiedTime;
 
-  public PartitionMetadata(Map<String, String> metadata, long creationTime, long lastUpdatedTime) {
+  public PartitionMetadata(Map<String, String> metadata, long creationTime, long lastModifiedTime) {
     this.metadata = Collections.unmodifiableMap(new HashMap<>(metadata));
     this.creationTime = creationTime;
-    this.lastUpdatedTime = lastUpdatedTime;
+    this.lastModifiedTime = lastModifiedTime;
```

Comment:
since we're into consistency of naming now.... call it lastModificationTime? similar to creationTime?